x-pack/plugins/security/public/management/roles/edit_role/edit_role_page.test.tsx

```diff
@@ -744,6 +744,29 @@ describe('<EditRolePage />', () => {
     expectSaveFormButtons(wrapper);
   });
 
+  it('render role with wildcard base privilege without edit/delete actions', async () => {
+    const wrapper = mountWithIntl(
+      <KibanaContextProvider services={coreStart}>
+        <EditRolePage
+          {...getProps({
+            action: 'edit',
+            role: {
+              name: 'my custom role',
+              metadata: {},
+              elasticsearch: { cluster: ['all'], indices: [], run_as: ['*'] },
+              kibana: [{ spaces: ['*'], base: ['*'], feature: {} }],
+            },
+          })}
+        />
+      </KibanaContextProvider>
+    );
+
+    await waitForRender(wrapper);
+
+    expect(wrapper.find('[data-test-subj="privilegeEditAction-0"]')).toHaveLength(0);
+    expect(wrapper.find('[data-test-subj="privilegeDeleteAction-0"]')).toHaveLength(0);
+  });
```

Comment:
Can we also add a check that this is rendered as read-only?

```suggestion
    expect(wrapper.find('[data-test-subj="privilegeDeleteAction-0"]')).toHaveLength(0);
    expectReadOnlyFormButtons(wrapper);
  });
```

Reply:
Sure, updated, thanks!